helper/src/main/java/me/lucko/helper/command/functional/FunctionalCommandBuilder.java

```diff
@@ -63,7 +63,7 @@
      * @return the builder instance
      */
     default FunctionalCommandBuilder<T> assertPermission(String permission) {
-        return assertPermission(permission, DEFAULT_NO_PERMISSION_MESSAGE);
+        return assertPermission(permission, null);
```

Comment:
Why remove the default message?

Reply:
I've removed default messages because, using the default Bukkit command permissions, if you don't set the permission message, it will use the default one of the server